src/appleseed.shaders/include/appleseed/color/as_color_transforms.h

```diff
@@ -749,7 +749,6 @@ color transform_CIELAB_to_CIEXYZ(
         get_illuminant_CIExyY(illuminant));
 }
 
-
```

Comment:
In source code, comment blocks of the form

    //
    // ...
    // ...
    //

should always be preceded by two blank lines.

It's the only case where we use two consecutive blank lines.

Reply:
ok, i wasn't sure about that, will fix now.